src/main/java/com/sonymobile/jenkins/plugins/mq/mqnotifier/MQConnection.java

```diff
@@ -64,8 +64,8 @@
     private Connection connection = null;
     private Channel channel = null;
 
-    private static LinkedBlockingQueue messageQueue;
-    private static Thread messageQueueThread;
+    private volatile LinkedBlockingQueue messageQueue;
```

Comment:
Not sure where we ended in this discussion, but volatile wasn't needed in the end, right?

Reply:
It wasn't when we talked about it, but removing it on the queue got me some more synchronization-related bugs from findbugs. I decided to go for good enough and leave it.